src/test/java/org/jabref/logic/importer/fetcher/AstrophysicsDataSystemTest.java

```diff
@@ -147,7 +142,7 @@ public void testGetName() {
     @Test
     public void searchByQueryFindsEntry() throws Exception {
         List<BibEntry> fetchedEntries = fetcher.performSearch("Diez slice theorem Lie");
-        assertEquals(Collections.singletonList(diezSliceTheoremEntry), fetchedEntries);
+        assertTrue(fetchedEntries.contains(diezSliceTheoremEntry));
```

Comment:
Please don't use assertTrue here to check for empty things. This makes it harder to debug in case of errors.

Reply:
The problem is that there is no guarantee that the endpoint contains exactly 1 or more results for the search.
And there is no assertContains or similiar method in Junit i know about.  
I appreciate any suggestion, how to write this test differently.

A fix should be commited to PR #5501, because this PR depends on the other